src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
If we statically add aliases and cmdlets for all core modules we can speed up PowerShell Core start-up.
@daxian-dbw Sorry for perseverance.
@lzybkr Could you please comment too?